doc/mergerepo_c.8

```diff
@@ -42,9 +42,9 @@ Show program\(aqs version number and exit
 .SS \-r \-\-repo REPOS
 .sp
 Repo url
-.SS \-a \-\-archlist ARCHLIST
+.SS \-\-repo\-prefix\-search ARCHLIST
 .sp
-Defaults to all arches \- otherwise specify arches
+Repository prefix to be replaced by repo\-prefix\-url. }, { repo\-prefix\-replace, 0, 0, G_OPTION_ARG_STRING, &(_cmd_options.repo_prefix_replace), Repository prefix URL by which the repo\-prefix is replaced. }, { archlist, \(aqa\(aq, 0, G_OPTION_ARG_STRING, &(_cmd_options.archlist), Defaults to all arches \- otherwise specify arches
```

Comment:
Crap, this looks broken... :(

Reply:
OK, what should I do about it? Should I drop the manpage patch?

Reply:
For now, I guess. Our only include the bit for the new option.